devtools/cli/src/test/java/io/quarkus/cli/CliProjectGradleTest.java

```diff
@@ -47,6 +55,20 @@ public static void allDone() {
         System.setProperty("user.dir", startingDir);
     }
 
+    private static void exportMavenLocalToGradleProperties(String localMavenRepo) throws Exception {
+        File gradleDir = new File(System.getProperty("user.home"), ".gradle");
+        if (!gradleDir.exists()) {
+            gradleDir.mkdir();
+        }
+        File propertyFile = new File(gradleDir, "gradle.properties");
+
+        Properties gradleProperties = new Properties();
+        gradleProperties.put("systemProp.maven.repo.local", localMavenRepo);
+        try (FileOutputStream os = new FileOutputStream(propertyFile)) {
+            gradleProperties.store(os, "Gradle property file");
```

Comment:
My understanding is that you change the global Gradle configuration of the user. That won't fly.

Reply:
yes, only, if the `maven.repo.local` system property is set. I can backup the user file and rollback to it after the test?

Reply:
> That won't fly.

+1! I have proxy settings in that file and I certainly don't expect Quarkus to modify this file.

Reply:
yes, it was definetely too aggressive. I reverted that part as it was not necessary.